spec/services/task_list_creator_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe TaskListCreator, type: :model do
+  let(:task_list_creator) { TaskListCreator.new }
+
+  let(:mock_workflow) { file_fixture("workflows/conversion.yml") }
+
+  before do
+    allow(YAML).to receive(:load_file).with("workflows/conversion.yml").and_return(
```

Comment:
You can use `. and_call_original` here instead of writing your own `and_return` to make the mock behave as the original would.

Reply:
I don't think this will work. I want to load the mock workflow defined in `./spec/fixtures/files/workflows/conversions.yml`. If I do `and_call_original`,  it will load the proper workflow (the on in `./workflows`)